view-binding-utils/src/main/java/com/airbnb/mvrx/viewbinding/FragmentViewBindingDelegate.kt

```diff
@@ -21,16 +24,18 @@ class FragmentViewBindingDelegate<T : ViewBinding>(
     private val bindMethod = bindingClass.getMethod("bind", View::class.java)
 
     init {
-        fragment.viewLifecycleOwnerLiveData.observe(fragment) { viewLifecycleOwner ->
-            viewLifecycleOwner.lifecycle.addObserver(object : LifecycleObserver {
-                @OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)
-                fun onDestroy() {
-                    // Lifecycle listeners are called before onDestroyView in a Fragment.
-                    // However, we want views to be able to use bindings in onDestroyView
-                    // to do cleanup so we clear the reference one frame later.
-                    clearBindingHandler.post { binding = null }
-                }
-            })
+        GlobalScope.launch(Dispatchers.Main) {
```

Comment:
can you share a github comment here about what this is fixing?

Also, global scope looks suspicious here - can you use the lifecycle of the fragment instead? `fragment.lifecycleScope.launch`

Reply:
yea tied to the lifecycle should work. lemme update

Reply:
added too description as well